### PR TITLE
feat(monitoring): count the number of processed documents

### PR DESCRIPTION
**Pull request by samsja**

### CODE REVIEW

File: jina/serve/executors/__init__.py [hunk 1]
```diff
@@ -250,10 +251,20 @@ async def __acall__(self, req_endpoint: str, **kwargs):
...
     async def __acall_endpoint__(self, req_endpoint, **kwargs):
         func = self.requests[req_endpoint]
-        if iscoroutinefunction(func):
-            return await func(self, **kwargs)
-        else:
-            return func(self, **kwargs)
+
+        _summary = (
+            self._summary_method.labels(
+                func.__name__, self.__class__.__name__, req_endpoint
```
Comment (jacobowitz) [inline jina/serve/executors/__init__.py:257]: I am not sure, but maybe we should remove the method name as a label now that we always also expose endpoint? Its kind of redundant now? I would think that endpoint/executor tuple should be good enough?

**Review by jacobowitz — COMMENTED** [review]
I just have one question, otherwise it looks good to me!

File: jina/serve/executors/__init__.py (hunk 1 above)
Comment (JoanFM) [reply jina/serve/executors/__init__.py:257]: Yes makes sense

Comment (samsja) [reply jina/serve/executors/__init__.py:257]: You are right I will do

**Review by jacobowitz — APPROVED** [review]
lgtm

**Pull request merged**